packages/@lwc/integration-karma/test/api/getComponentDef/index.spec.js

```diff
@@ -152,6 +156,60 @@ describe('@api', () => {
             childMethod: PublicMethodsInheritance.prototype.childMethod,
         });
     });
+
+    it('should log warning when accessing a private prop', () => {
+        const elm = createElement('x-private-accessor', { is: PrivateAccessors });
+        document.body.appendChild(elm);
+
+        expect(() => {
+            elm['privateProp'];
+        }).toLogWarningDev(message('privateProp'));
+    });
+
+    it('should log warning when setting a private prop', () => {
+        const elm = createElement('x-private-accessor', { is: PrivateAccessors });
+        document.body.appendChild(elm);
+
+        expect(() => {
+            elm['privateProp'] = 'foo';
+        }).toLogWarningDev(message('privateProp'));
+    });
+
+    it('should not log warning when accessing a public prop', () => {
+        const elm = createElement('x-private-accessor', { is: PrivateAccessors });
+        document.body.appendChild(elm);
+
+        expect(() => {
+            elm['publicProb'];
+        }).not.toLogWarningDev(message('publicProp'));
```

Comment:
```suggestion
        }).not.toLogWarningDev();
```

No argument needed in the case of `not`.